### PR TITLE
DATAREDIS-873 - Add possibility to use single collection parameter to SetOperations methods

### PR DESCRIPTION
**Pull request by Dominys**

### CODE REVIEW

File: src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2011-2018 the original author or authors.
+ possibilityt 2011-2018 the original author or authors.
```
Comment (mp911de) [inline src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java]: Please do not change license headers.

File: src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
  * @author Costin Leau
  * @author Christoph Strobl
  * @author Mark Paluch
+ * @author Dominys
```
Comment (mp911de) [inline src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java]: Thanks for adding `@author` tags. Please use your actual name, not the GitHub handle or nickname.

File: src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java (hunk 1 above)
Comment (Dominys) [reply src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java]: @mp911de Sorry)) Did not check that

File: src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java (hunk 2 above)
Comment (Dominys) [reply src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java]: @mp911de Ok. I'll change this.

**Comment by Dominys** [comment]
@mp911de Hi! Yes, I can align ReactiveSetOperations too.

**Comment by Dominys** [comment]
@mp911de Done

**Comment by mp911de** [comment]
Thanks a lot. That's merged and polished now.

**Pull request closed without merging**